src/coreComponents/unitTests/meshTests/testVTKImport.cpp

```diff
@@ -91,60 +91,99 @@ TEST( VTKImport, cube )
     ASSERT_EQ( cellBlockManager.numFaces(), expected( 108, { 75, 42 } ) );
 
     // The information in the tables is not filled yet. We can check the consistency of the sizes.
-    ASSERT_EQ( cellBlockManager.getNodeToFaces().size(), expectedNumNodes );
-    ASSERT_EQ( cellBlockManager.getNodeToElements().toCellIndex.size(), expectedNumNodes );
+    ASSERT_EQ( cellBlockManager.getNodeToFaces().size(), expectedNumNodes ); //64
+    ASSERT_EQ( cellBlockManager.getNodeToElements().toCellIndex.size(), expectedNumNodes ); //64
 
     // We have all the 4 x 4  x 4 = 64 nodes in the "all" set.
-    SortedArray< localIndex > const & allNodes = cellBlockManager.getNodeSets().at( "all" );
+    SortedArray< localIndex > const & allNodes = cellBlockManager.getNodeSets().at( "all" ); //64
     ASSERT_EQ( allNodes.size(), expectedNumNodes );
 
     // The "2" set are all the boundary nodes (64 - 8 inside nodes = 56),
     // minus an extra node that belongs to regions -1 and 9 only.
-    SortedArray< localIndex > const & nodesRegion2 = cellBlockManager.getNodeSets().at( "2" );
-    ASSERT_EQ( nodesRegion2.size(), expected( 55, { 39, 27 } ) );
-
-    // Region "9" has only one quad, on the greater `x` direction.
-    // This hex will belong to MPI rank 1.
-    SortedArray< localIndex > const & nodesRegion9 = cellBlockManager.getNodeSets().at( "9" );
-    ASSERT_EQ( nodesRegion9.size(), expected( 4, { 0, 4 } ) );
+    std::cout << "> NodeSets : \n";
```

Comment:
the "`cout`" are just for debugging purposes? If so, please remove them, else they will pollute the unit test output